src/common/components/Navbar.vue

```diff
@@ -0,0 +1,408 @@
+              template(slot="header" v-if="getActiveMenu.title")
+                span {{ getActiveMenu.title }}
+
+              template
+                section.navbar__dropdown-content(v-if="getActiveMenu.type === 'notification'")
+                  .navbar__notification
+                    .notification-item(role="button" v-for="t in 20" class="pt-16")
+                      .notification-item__wrapper
+                        .notification-item__title(aria-label="Congrats! You got 5 DBIO!") Congrats! You got 5 DBIO!
+                        .notification-item__description(aria-label="Congrats! You got 5 DBIO!") 2 hour ago
+
+                section.navbar__dropdown-content(v-if="getActiveMenu.type === 'settings'")
+                  .navbar__settings
+                    .settings-item(role="button")
+                      .settings-item__wrapper
+                        .settings-item__title(aria-label="Profile") Profile
+                        ui-debio-icon.settings-item__icon(:icon="userIcon" size="24" stroke color="#C400A5")
+                    .settings-item(role="button")
+                      .settings-item__wrapper
+                        .settings-item__title(aria-label="Profile") Switch Role
+                        ui-debio-icon.settings-item__icon(:icon="usersIcon" size="24" stroke color="#C400A5")
+                    .settings-item(role="button")
+                      .settings-item__wrapper
+                        .settings-item__title(aria-label="Profile") Logout
+                        ui-debio-icon.settings-item__icon(:icon="logoutIcon" size="24" stroke color="#C400A5")
+
+                section.navbar__dropdown-content(v-if="getActiveMenu.type === 'polkadot' || getActiveMenu.type === 'metamask'")
+                  ui-debio-input(label="Your Address" ref="polkadot" disabled value="0xab5801a7d3983ab5U29rmtY794Heuerk51b.." block)
+                    ui-debio-icon(
+                      slot="icon-append"
+                      :icon="copyIcon"
+                      view-box="0 0 40 40"
+                      stroke
+                      color="#5640A5"
+                      size="20"
+                      role="button"
+                      @click="handleCopy('0xab5801a7d3983ab5U29rmtY794Heuerk51b..')"
+                    )
+                  .navbar__balance
+                    .navbar__balance-wrapper
+                      span.navbar__balance-type {{ getActiveMenu.currency }} Balance
+                      span.navbar__balance-amount 100
+                    .navbar__balance-usd (100 USD)
+
+              template(slot="footer" v-if="getActiveMenu.action")
+                v-btn.navbar__footer-button(block color="primary" outlined) {{ getActiveMenu.action }}
+</template>
+
+<script>
+import {
+  searchIcon,
+  bellIcon,
+  settingIcon,
+  userIcon,
+  usersIcon,
+  logoutIcon,
+  polkadotIcon,
+  metamaskFoxIcon,
+  copyIcon
+} from "@/core/icons"
```

Comment:
I don't see a directory named `core/icons` did you forgot to push this?

Reply:
I see, so you have this on a separate PR? Next time please just merge it as one, it's okay if you want to PR multiple features at once, as long as you put the issue link in the description.